src/include/parser/parse_coerce.h

```diff
@@ -137,4 +137,7 @@ typedef int32 (*select_common_typmod_hook_type) (ParseState *pstate, List *exprs
 typedef Node *(*handle_constant_literals_hook_type) (ParseState *pstate, Node *e);
 extern PGDLLEXPORT handle_constant_literals_hook_type handle_constant_literals_hook;
 
+typedef void (*set_common_typmod_case_expr_hook_type) (ParseState *pstate, List *exprs, CaseExpr *newc, int32 typmod);
+extern PGDLLIMPORT set_common_typmod_case_expr_hook_type set_common_typmod_case_expr_hook;
```

Comment:
Instead of `PGDLLIMPORT`, it should be `PGDLLEXPORT`.

Reply:
Good catch

Reply:
As discussed by rishab it should be PGDLLIMPORT